src/main/java/io/cdap/plugin/format/MultiTableConf.java

```diff
@@ -146,7 +151,14 @@ public String getTableNameField() {
   }
 
   @Nullable
-  public String getWhereClause() { return whereClause; }
+  public Integer getNumSplits() {
```

Comment:
a better name is splitsPerTable, this sounds like the total number of splits

Reply:
done